packages/rum/test/benchmarks/analyzer.js

```diff
@@ -181,12 +222,57 @@ function capturePayloadInfo(payload) {
   }
 }
 
+function getMemoryAllocationPerFunction({ profile }) {
+  const allocations = []
+  /**
+   * Whitelist built functions from the sample
+   */
+  const whitelist = [
```

Comment:
I think you mean blacklist here, but even then I think something like `excludeFunctionNames` is better.